core/client/kt/verify.go

```diff
@@ -111,15 +109,6 @@ func (v *Verifier) VerifyGetEntryResponse(ctx context.Context, userID string, in
 	}
 	Vlog.Printf("✓ Signed Map Head signature verified.")
 
-	// Verify SCT.
-	sct := new(ct.SignedCertificateTimestamp)
-	if _, err := tls.Unmarshal(in.SmhSct, sct); err != nil {
-		return err
-	}
-	if err := v.log.VerifySCT(ctx, in.GetSmh(), sct); err != nil {
-		Vlog.Printf("✗ Signed Map Head CT inclusion proof verification failed.")
-		return err
-	}
-	Vlog.Printf("✓ Signed Map Head CT inclusion proof verified.")
+	// TODO: Verify inclusion proof.
```

Comment:
This reduces the security of the system, do you know when we're planning on adding this verification step?